sensor/Guardian.js

```diff
@@ -103,9 +105,23 @@ module.exports = class {
   }
 
   async handlLegacy() {
+    // box might be removed from msp but it was offline before
+    // remove legacy settings to avoid the box been locked forever
+    const mspResult = await this.checkBoxWithMsp();
+    if (mspResult.is_member === true) return;
+    const cloudResult = await this.checkBoxWithCloud();
+    if (cloudResult.is_member === true) return;
+
+    if ((mspResult.is_member === false && cloudResult.is_member === false) || (
```

Comment:
Will this be triggered if the internet cable is disconnected on the box? If yes, may need to handle the error triggered by connection failure.

Reply:
no, if internet is disconnected, both of this will be `undefined`, that't why I used `===`